### ISSUE TITLE
Docs about color mode doesn't work

### ISSUE THREAD

**Issue by svandeneertwegh** [django-cms/django-cms#7484]
## Description
Problem is that fixed color scheme doesnt work! Also set with the below variables:

```
CMS_COLOR_SCHEME = "light"
CMS_COLOR_SCHEME_TOGGLE = True
```
set in the `settings.py`

## Additional information (CMS/Python/Django versions)
```
Django==4.1.5
Django-cms==4.1.0rc1
Python 3.10
```

* [x] Yes, I want to help fix this issue and I will join #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to confirm with the community that a PR is welcome. 
* [ ] No, I only want to report the issue.

**Comment by marksweb** [comment]
I don't think this feature has been ported to version 4

**Comment by fsbraun** [comment]
Nope, it has not. It is a major refactoring of the css. A solution would be to cherry-pick the corresponding merges into to `develop` branch for `develop-4`.

On first sight I see the following merges (from end to front): `fcfe77f637253d0359221ca51d27232742e7ef57`, `a7f8cd44f2394d5ca78e55244322959601b22a2d`, `abc6e6c5bb93b3b92f5ac9d2b185ddb597f162e7`, `e7c1f3f6393cdb393a3f867ef5c4fced4dee81c7`, `05b6d834425b549537deee1af4075821a8fe6af4`, `b2d9a088b5625753a5673be30450d978d9d9a357`

**Comment by fsbraun** [comment]
fixed in django-cms 4.1.0rc4
